assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -51,6 +52,15 @@ const StripePopUpButton = (props: PropTypes) => (
   />
 );
 
+function formIsValid(formClassName: string) {
+  const form = document.getElementsByClassName(formClassName);
+  let isValid = true;
+  if (form && form.length > 0) {
+    const formElements = [...form[0].getElementsByTagName('input')];
+    isValid = formElements && formElements.reduce((acc, el) => acc && el.validity.valid, true);
```

Comment:
you don't need the `formElements &&` here

Reply:
`formElements.every(el => el.validity.valid)` might be clearer